Dockerfile

```
@@ -0,0 +1,9 @@
+FROM golang:1.17
+
+WORKDIR /go/src/walrus
```

I prefer just `/app` as you can see in this [example](https://docs.docker.com/language/golang/build-images/#multi-stage-builds)
```suggestion
WORKDIR /app
```